build.sh

```diff
@@ -2,14 +2,32 @@
 
 if command -v dpkg-query -l zip
 then
-  composer install -a -o --no-dev
+  set -e
+
+  if ! command -v zip
+  then
+    echo "Zip package is not currently installed"
+    exit
+  fi
+
+  if ! command -v php7.0
```

Comment:
Composer states that the package should be compatible with PHP 5.6: `"php": "^5.6 || ^7.0"`